.github/workflows/ci.yml

```diff
@@ -95,7 +95,7 @@ jobs:
       aws_region: ${{ inputs.aws_region }}
       ci_cli_version: ${{ inputs.ci_cli_version }}
       earthly_version: ${{ inputs.earthly_version }}
-      target: check
+      target: check-*
```

Comment:
I think it would be better to specify a target name just as `check`, `build`, `test` etc, without any special characters at this place.

Reply:
No, we don't want this, as it breaks backward compatibility. By specifying a `-*` we are indicating we want to capture more than a single target. We don't want to enforce capturing multiple targets with no way to go back to the original form.

Reply:
Got it, I see, thanks for explanation